public/locales/en.json

```diff
@@ -93,8 +93,7 @@
   "liquidityMining.header": "Liq. mining",
   "liquidityMining.title": "Liquidity mining",
   "liquidityMining.action.create": "CREATE LIQ. MINING",
-  "liquidityMining.create.title": "Create a liquidity mining pool",
-  "liquidityMining.create.subtitle": "Follow these simple steps to preview and create a liquidity mining pool on swapr",
+  "liquidityMining.create.title": "Create a Rewards Campaign",
```

Comment:
We should refactor `en.json` and create scoped files. I love that you're going in this direction with dotted notation keys.